zql/Makefile

```diff
@@ -9,9 +7,15 @@ run: all
 
 .PHONY: zql.go
 zql.go:
-	cpp -DGO -E -P zql.peg | $(PIGEON) -o $@
-	$(GOIMPORTS) -w $@
+	mkdir -p $(deps)
+	echo 'module deps' > $(deps)/go.mod
+	cd $(deps) && GOBIN=$(deps)/bin go get \
+		github.com/mna/pigeon@v1.0.1-0.20190520151103-9fec3898cef8 \
+		golang.org/x/tools/cmd/goimports@v0.0.0-20200204192400-7124308813f3
+	cpp -DGO -E -P zql.peg | $(deps)/bin/pigeon -o $@
+	$(deps)/bin/goimports -w $@
 
 .PHONY: zql.js
 zql.js:
-	cpp -E -P zql.peg | $(PEGJS) -o $@
+	npm install --global --prefix $(deps) pegjs@0.10.0
```

Comment:
-1 on this, I don't think building should have side effects outside of the source tree...

Reply:
derp, overlooked the --prefix bit